packages/bento-design-system/src/Avatar/createAvatar.tsx

```diff
@@ -35,7 +50,15 @@ export function createAvatar(config: AvatarConfig) {
           borderRadius={config.radius}
           boxShadow={config.outline}
         >
-          {initial ? (
+          {imageSrc && imageAvailable !== false ? (
+            <img
+              style={{ display: imageAvailable ? "block" : "none" }}
+              src={imageSrc}
+              className={avatarImage({})}
+              onLoad={handleImageLoaded}
+              onError={handleImageErrored}
+            />
```

Comment:
I propose we use `<Box as="img">` here, which also gives you the ability to use `display` directly (`<Box display={imageAvailable ? "block" : "none"} />`)